roles/softserve/tasks/main.yml

```diff
@@ -0,0 +1,81 @@
+---
+- name: Create a user
+  user:
+    name: "{{ user }}"
+    state: present
+
+- name: Install required packages
+  package:
+    state: present
+    name: "{{ item }}"
+  with_items:
+  - python-virtualenv
+  - gcc
+  - git
+  - python-devel
+  - postgresql-devel
+  - mock
+
+- name: Create the project directory
+  file:
+    path: "{{ destination }}"
+    state: directory
+    owner: softserve
+    group: softserve
+    recurse: yes
+
+- name: Clone the code
+  git:
+    repo: 'https://github.com/gluster/softserve.git'
+    dest:  '{{ destination }}/code'
+    version: HEAD
+    update: yes
+    force: yes
+
+- name: Add tmpfiles snippet
+  template:
+    dest: /etc/tmpfiles.d/softserve.conf
+    src: softserve.tmpfile.conf
+  notify: Create systemd tmpfile
+
+- name: Add gunicorn systemd service
+  template:
+    dest: /etc/systemd/system/softserve.service
+    src: softserve.service
+  notify: Reload systemd
```

Comment:
So, we need the meta flush_handler here, because otherwise, we will started the service before systemd is reloaded and before the tpmfile is created. Sorry if I wasn't clear on previous comments :/